src/bci_build/package/golang/README.md.j2

```diff
@@ -4,27 +4,55 @@
 
 ## Description
 
-[Go](https://go.dev/) (a.k.a., Golang) is a statically-typed programming language, with syntax loosely derived from C. Go offers additional features such as garbage collection, type safety, certain dynamic-typing capabilities, additional built-in types (for example, variable-length arrays and key-value maps) as well as a large standard library.
+[Go](https://go.dev/) (a.k.a., Golang) is a statically-typed programming
+language, with syntax loosely derived from C. Go offers additional features
+such as garbage collection, type safety, certain dynamic-typing capabilities,
+additional built-in types (for example, variable-length arrays and key-value
+maps) as well as a large standard library.
+
+{%- if image.version.endswith('-openssl') %}
+
+## FIPS 140-3
+
+The image includes a FIPS 140-2/140-3 enabled Go wrapper that prefers using OpenSSL
+for cryptographic operations, if available at runtime.
+This allows the use of FIPS 140-2/140-3 validated routines, provided by the OpenSSL
+library, for cryptographic operations in the container environment.
+
+{%- endif %}
+
 
 ## Usage
+There are two ways to work with Go images. First, you can encapsulate your
+application in a `scratch` container image, which is essentially an empty
+image. This approach will not function if your Go application depends on libc
+or any other library, as they will not be available.
+
+A second method is to use a slim base container image with just the minimal
```

Comment:
The second method is not described here? Or is it the SLE BCI container? If so, please consider changing the docker file description of RUN CGO_ENABLED=0, as if I got it right, that should be needed only if building FROM scratch.